circle/src/pcs.rs

```diff
@@ -0,0 +1,146 @@
+use alloc::vec::Vec;
+
+use itertools::izip;
+use p3_commit::{DirectMmcs, OpenedValues, Pcs};
+use p3_field::extension::ComplexExtendable;
+use p3_field::ExtensionField;
+use p3_matrix::dense::{RowMajorMatrix, RowMajorMatrixView};
+use p3_matrix::routines::columnwise_dot_product;
+use p3_matrix::{Matrix, MatrixRows};
+use p3_util::log2_strict_usize;
+use tracing::instrument;
+
+use crate::cfft::Cfft;
+use crate::domain::CircleDomain;
+use crate::util::{univariate_to_point, v_n};
+
+pub struct CirclePcs<Val, InputMmcs> {
+    pub log_blowup: usize,
+    pub cfft: Cfft<Val>,
+    pub mmcs: InputMmcs,
```

Comment:
`mmcs` is for Merkle-lized commitment scheme?

Reply:
I was just guessing the naming convention here. So none of the two `m` is for "Merkle"?